cmd/dockerd/daemon.go

```diff
@@ -264,11 +265,22 @@ func (cli *DaemonCli) start(opts daemonOptions) (err error) {
 	// Notify that the API is active, but before daemon is set up.
 	preNotifySystem()
 
-	d, err := daemon.NewDaemon(cli.Config, registryService, containerdRemote)
+	pluginStore := plugin.NewStore(cli.Config.Root)
```

Comment:
`plugin.NewManager` takes care of that. Infact, we dont need to pass down `cli.Config.Root` while initializing store anymore. I'll remove this.